third_party/clang_llvm_x86_64_linux_gnu_ubuntu.BUILD

```diff
@@ -48,11 +48,28 @@ filegroup(
 
 cc_toolchain_import(
     name = "llvm_libunwind",
-    hdrs = ["lib/clang/12.0.0/include/unwind.h"],
-    includes = ["lib/clang/12.0.0/include"],
+    hdrs = glob(["lib/clang/*/include/unwind.h"]),
+    includes = ["lib/clang/15.0.6/include"],
```

Comment:
As a bit of a comprimise you might be able to add a variable at the top of this build file and then just use `.format()` to allow for easy future updates. So something like this...
```python
CLANG_VERSION = "15.0.6",

cc_toolchain_import(
  name = "llvm_unwind",
  includes = ["lib/clang/{clang_version}/include".format(clang_version=CLANG_VERSION)],
  # ....
)
```

Reply:
Done!